packages/discovery-provider/ddl/migrations/0145_create_shares_table.sql

```diff
@@ -0,0 +1,42 @@
+begin;
+
+-- Add share_count column to aggregate_track table
+ALTER TABLE aggregate_track ADD COLUMN IF NOT EXISTS share_count int DEFAULT 0;
+
+-- Add share_count column to aggregate_playlist table
+ALTER TABLE aggregate_playlist ADD COLUMN IF NOT EXISTS share_count int DEFAULT 0;
+
+-- Add track_share_count column to aggregate_user table
+ALTER TABLE aggregate_user ADD COLUMN IF NOT EXISTS track_share_count int DEFAULT 0;
+
+-- Create sharetype enum similar to reposttype and savetype
+DO $$ BEGIN
+    CREATE TYPE public.sharetype AS ENUM (
+        'track',
+        'playlist',
+        'album'
```

Comment:
we only use `track` and `playlist` in saves and reposts, FWIW
![image](https://github.com/user-attachments/assets/f58425ea-73a7-494d-92d5-0613229dde49)

Reply:
Oh interesting. They have a third value in their enums so I was just duplicating that. I can remove it from shares. We can always add it back in later.